third_party/amd/backend/compiler.py

```diff
@@ -113,6 +113,9 @@ def parse_options(self, opts) -> Any:
                 supported_fp8_dtypes.update({'fp8e4nv', 'fp8e4b8', 'fp8e5b16'})
             elif self.target.arch == 'gfx950':
                 supported_fp8_dtypes.update({'fp8e4nv', 'fp8e5'})
+            elif 'gfx12' in self.target.arch:
+                # Triton does not support f8 dtypes for gfx12 target yet.
+                supported_fp8_dtypes.update({})
```

Comment:
What about making the default in `HIPOptions` as empty and set the proper ones for each arch in the above so we don't need to explictly mention an arch does not support any yet?